README.md

```diff
@@ -1,11 +1,6 @@
 # 📰 My Portfolio
```

Comment:
:red_circle:  Please make sure your readme includes the table of content as it is in  [the Microverse template](https://github.com/microverseinc/readme-template)